olive/workflows/run/__main__.py

```diff
@@ -3,21 +3,39 @@
 # Licensed under the MIT License.
 # --------------------------------------------------------------------------
 import argparse
+import sys
+from pathlib import Path
 
 from olive.common.utils import set_tempdir
 from olive.workflows import run
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser("Olive Workflow: Custom Run")
-    parser.add_argument("--config", type=str, help="Path to json config file", required=True)
+    parser.add_argument("--olive-config", type=str, help="Path to Olive (json) config file", required=False)
+    parser.add_argument("--run-config", "--config", type=str, help="Path to json config file", required=True)
```

Comment:
Description and name should be clearer that to indicate that this optional .json file, meant for advanced users only, lists location of passes you want to use in your --config.

Reply:
Are you referring to using something else in place of 'olive-config'? I started off with calling it passes-config but then realized there are other scattered input files that can be merged into one and that can serve as a more *entire module config* rather than having to explain to user what each config does.

Reply:
Added descriptive help text.

Reply:
If we change the `--config`, all the examples' readme need to be updated. Otherwise, better to keep using the original parameter name.

Reply:
`--config` was marked deprecated not removed. So using `--config` was still acceptable but `--run-config` was preferred. Regardless, that change has been completed reverted. `--config` stays and the new config parameter is called `--package-config`.